src/patterns/PaymentForm.vue

```diff
@@ -0,0 +1,288 @@
+      <TextInput
+        v-model="firstName"
+        label="First Name"
+        :class="b('first-name').toString()"
+        required />
+      <TextInput
+        v-model="lastName"
+        label="Last Name"
+        :class="b('last-name').toString()"
+        required />
+      <TextInput
+        v-model="zipCode"
+        label="Zip Code"
+        type="number"
+        :class="b('zip-code').toString()"
+        :validations="zipValidator"
+        :error-messages="{
+          zipValidator: 'Invalid Location',
+          required: 'Invalid Location'
+        }"
+        required />
+      <VSpacer size="xxs" />
+      <TextInput
+        id="card-number"
+        ref="cardNumber"
+        :class="b('card-number').toString()" />
+      <TextInput
+        id="card-expiry"
+        ref="cardExpiry"
+        :class="b('card-expiry').toString()" />
+      <TextInput
+        id="card-cvc"
+        ref="cardCvc"
+        :class="b('card-cvc').toString()" />
+    </div>
+    <div
+      id="card-errors"
+      role="alert" />
+    <div
+      v-if="showError"
+      :class="b('error-text').toString()">
+      <Paragraph
+        font-size="m"
+        font-weight="light">
+        There was an error processing your payment, please refresh and try again.
+      </Paragraph>
+    </div>
+    <VSpacer size="xl" />
+  </div>
+</template>
+
+<script>
+/* global Stripe */
+// import tokens from '@/assets/tokens/tokens.json';
+import { zipValidator } from '../utils/validators/location-vuelidate';
+
+/**
+ * Payment Form
+ */
+
+export default {
+  name: 'PaymentForm',
+  status: 'under-review',
+  release: '1.0.0',
+  blockName: 'payment-form',
+  props: {
+    email: {
+      type: String,
+      default: '',
+    },
+    paymentError: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  data() {
+    return {
+      firstName: '',
+      lastName: '',
+      zipCode: '',
+      showError: false,
+    };
+  },
+
+  computed: {
+    zipValidator() {
+      return {
+        zipValidator,
+      };
+    },
+  },
+
+  watch: {
+    paymentError(bool) {
+      if (bool === true) this.showError = true;
+    },
+  },
+
+  created() {
+    /**
+     * Append the Stripe script to the head of our HTML
+     * (Stripe only supports it being in the head). Wait for
+     * the script to finish loading and then mount the Stripe
+     * Elements
+     */
+    const stripeScript = document.createElement('script');
+    document.head.appendChild(stripeScript);
+    stripeScript.onload = () => {
+      this.mountStripe();
+    };
+    stripeScript.setAttribute('src', 'https://js.stripe.com/v3/');
+    /**
+     * Get stripe key
+     *
+     * @syscall stripe/key
+     * @returns {{stripeKey: String}}
+     */
+    this.$syscall('config/stripe/key')
+      .then(response => {
+        this.stripeKey = response;
+      });
+  },
+
+  mounted() {
+  },
+
+  methods: {
+    mountStripe() {
+      this.stripe = Stripe(this.stripeKey);
```

Comment:
Could you document in this pattern's description that it's expecting the Stripe library to be loaded into the Parent environment?